### DISCUSSION TITLE
scvelo.pl.velocity scale of gene expression plot

### DISCUSSION THREAD

**Discussion by jinyoung5484** [theislab/scvelo#514] [Q&A]
Hello,

I have a question about scv.pl.velocity() plotting method.
When I plot certain genes with scv.pl.velocity(), I see gene expression plot on the right side, but scales vary a lot by genes.
How do you guys scaling the gene expression on this plot, or what do you guys use to scale gene expression?

Thank you.

**Comment by WeilerP** [comment]
You can check out the [docs](https://scvelo.readthedocs.io/) for some tutorials. [This](https://scvelo.readthedocs.io/VelocityBasics/) one, for example, briefly discusses the basics of RNA velocity (including preprocessing). In general, genes are filter based on their counts, CPM normalized, log transformed and the most highly variable genes selected. Following, counts are imputed by first moments over the k nearest neighbors.
